### ISSUE TITLE
README incorrect docs the starting data block

### ISSUE THREAD

**Issue by mallochine** [MatheusrdSantos/vue-quick-chat#9]
What a shame this took me an hour to debug. My software skills are not what they used to be.

You have 'messagesDisplay' in the wrong code block

INCORRECT
```
        message:{
          myself: {
            bg: '#fff',
            text: '#bdb8b8'
          },
          others: {
            bg: '#fb4141',
            text: '#fff'
          }
        },
        messagesDisplay: {
            bg: '#f7f3f3'
        },
```

CORRECT
```
        message:{
          myself: {
            bg: '#fff',
            text: '#bdb8b8'
          },
          others: {
            bg: '#fb4141',
            text: '#fff'
          },
          messagesDisplay: {
              bg: '#f7f3f3'
          },
        },
```

**Comment by mallochine** [comment]
I am too lazy to submit pull request. I apologize. Back to work for me.

**Comment by MatheusrdSantos** [comment]
Hi, @mallochine.
I fixed this issue here #10, so I'll close it.
Thanks for reporting this issue.
